### ISSUE TITLE
v4.5.0 does not work because of missing assets

### ISSUE THREAD

**Issue by mikkeloscar** [squidfunk/mkdocs-material#1363]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

Expected assets are not included in `v4.5.0`

Output of `mkdocs serve -s`

```
[I 191202 15:01:31 server:296] Serving on http://127.0.0.1:8000
[I 191202 15:01:31 handlers:62] Start watching changes
[I 191202 15:01:31 handlers:64] Start detecting changes
[W 191202 15:01:36 web:2242] 404 GET /assets/stylesheets/application.82a18eb7.css (127.0.0.1) 3.40ms
[W 191202 15:01:36 web:2242] 404 GET /assets/javascripts/application.d151b584.js (127.0.0.1) 10.49ms
[W 191202 15:01:36 web:2242] 404 GET /assets/javascripts/application.d151b584.js (127.0.0.1) 12.61ms
[I 191202 15:01:37 handlers:135] Browser Connected: http://127.0.0.1:8000/
```

### Expected behavior

Assets should be included (they are included in master since https://github.com/squidfunk/mkdocs-material/commit/aa716ae3babaed7339e53d72d396ed26a1287763)

I don't think I need to add more information, the assets are simply missing so it would be helpful if you could cut a new release :)

Fwiw I'm using this package of `mkdocs-material`: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=mkdocs-material

**Comment by squidfunk** [comment]
Yeah, you're right. 4.5.0 Doesn't include the assets. I think there was something wrong with the build. Will release 4.5.1 shortly which should fix things.

**Comment by squidfunk** [comment]
Fixed in 4.5.1. Thanks for reporting!

**Comment by mikkeloscar** [comment]
Yup, this fixed the problem. Thanks a lot!
